tests/test_edgeql_ddl.py

```diff
@@ -608,7 +608,7 @@ async def test_edgeql_ddl_16(self):
             CREATE TYPE TestSelfLink3 {
                 CREATE PROPERTY foo3 -> std::str;
                 CREATE PROPERTY bar3 -> std::str {
-                    SET default := TestSelfLink3.foo3;
+                    SET default := .foo3;
                 };
             };
```

Comment:
The fix broke this test.

Turns out if you tried to do an INSERT after this test, it would fail. So it was wrong before.